### PR TITLE
data:mjr2595

### PR DESCRIPTION
**Pull request by mjr2595**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3999"></a>

### CODE REVIEW

**Review by iamthecloverly — APPROVED** [review]

**Review by stevenovak123 — APPROVED** [review]
LGTM, Thank you

File: data/mjr2595.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "links": [
+    {
+      "name": "Git over here",
+      "url": "https://github.com/mjr2595",
+      "icon": "FaGithubAlt"
+    },
+    {
+      "name": "Link with me",
+      "url": "https://www.linkedin.com/in/mjraymond",
+      "icon": "FaLinkedinIn"
```
Comment (Panquesito7) [inline data/mjr2595.json:16]: Wrong icon name.
```suggestion
      "icon": "FaLinkedin"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/mjr2595.json (hunk 1 above)
Comment (mjr2595) [reply data/mjr2595.json:16]: Hi @Panquesito7! 

I'm seeing "FaLinkedinIn" as an available icon when searching https://linkfree.eddiehub.io/icons

image

I'm ok with making this change if this alt icon isn't really available. Although I would also then post an issue 😄.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**